### PR TITLE
Create vuetify project

### PR DESCRIPTION
**Pull request by lesser1**
Please note, that I sort of accidentally committed the "Essentials" Vuetify preset. This includes an example application saying something like "Welcome to Vuetify 3". It comes with a HelloWorld Vue component. Of course, these "demo files" will soon be deleted.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,70 @@
 # frontend
 The frontend of the MiSArch reference architecture: The web application for the online shop.
+
+## Project setup
```
Comment (nk-coding) [inline README.md:4]: as you decided to use yarn, please adapt README

**Review by nk-coding — CHANGES_REQUESTED** [review]
I'm not  sure if you already want to remove template stuff with this PR.
I usually do, but I would also see reasons to do this in a separate PR

File: README.md (hunk 1 above)
Comment (lesser1) [reply README.md:4]: Will do

**Comment by lesser1** [comment]
> I'm not sure if you already want to remove template stuff with this PR. I usually do, but I would also see reasons to do this in a separate PR

See my comment at the start of this PR's conversation (Yes, I added the comment after you commented and yes, I managed to not see your comment, my bad). Anyway, since I am fairly new to Vue I would like to keep the stuff for now in order to not mess anything up.

**Review by nk-coding — APPROVED** [review]

**Pull request merged**